packages/design-system-twrnc-preset/src/Theme/Theme.utilities.ts

```diff
@@ -1,7 +1,17 @@
-import baseConfig from '../../tailwind.config';
 import type { ColorSet, ColorScheme } from '../twrnc-settings';
 import { colorSetList, typographyTailwindConfig } from '../twrnc-settings';
 
+// Inline base Tailwind config to avoid relative import issues when package is consumed
+const baseConfig = {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  theme: {
+    extend: {
+      colors: {},
+    },
+  },
+  plugins: [],
+};
+
```

Comment:
Do we even need this? It's essentially empty

Reply:
I remember it was needed for the script